.gitignore

```diff
@@ -43,6 +43,8 @@ share/python-wheels/
 .installed.cfg
 *.egg
 MANIFEST
+requirements/node_modules
+.idea
```

Comment:
I prefer not to add text editor- or OS-specific ignores since there are thousands and it isn't clear which to add. Users should add those themselves in their global gitignore.